lib/src/controller/feed.dart

```diff
@@ -13,6 +13,7 @@ class FeedInput with _$FeedInput {
   const factory FeedInput({
     required String name,
```

Comment:
I just realized this property is redundant right here, since the name is already contained in the database's key, similarly to profiles, accounts, servers, and filter lists. Can it be removed?